src/Entity/HomeBlock.php

```diff
@@ -158,83 +158,69 @@ public function getPosition(): ?int
         return $this->position;
     }
 
-    public function setPosition(int $position): self
+    public function setPosition(int $position): void
     {
         $this->position = $position;
-
-        return $this;
```

Comment:
Method calls chaining in Doctrine just sucks! There are no reasons for returning `$this` in setter methods just to provide a fluent interface. So better to remove them as done in this PR and update locations where methods are chained.